app/views/comments/_single_comment.html.erb

```diff
@@ -59,8 +59,10 @@
         <% end %>
       <% end %><!-- end caching -->
 
+      <% if logged_in_as_admin? %>
+        <p class="comment ip"><%= ts("IP Address: %{ip_address}", ip_address: single_comment.ip_address.blank? ? "No address recorded" : single_comment.ip_address) %></p>
```

Comment:
I can see my issue on dev is strangeness in how nginx is serving a really old file.